### ISSUE TITLE
Cannot search by attachments' mimetype under supporting files

### ISSUE THREAD

**Issue by hyebin-bina** [huridocs/uwazidocs#4816]
**Describe the bug**
Uwazi search box used to allow the users to filter entities by their supporting files' mime-type but it does not work anymore. 

**To Reproduce**
1. When typing the query in the search box as follows: attachments.mimetype:“image/png” 
2. It filters entities that have the text "image," instead of the entities that have png supporting files. (Please see the screenshot attached.)

**Expected behavior**
As this search hack worked when I tested it in mid-May, I assume something within the search has changed. 

**Screenshots**
![Screen Shot 2022-07-04 at 10 02 26 PM](https://user-images.githubusercontent.com/85096417/177160883-0f023680-7879-4392-9b37-e422cd7dacd1.png)

**Device (please select all that apply)**
 - [V] Desktop

 - [ ] Mobile

**Browser**
Chrome

**Additional context**
Sidenote: Further, the mime-type search is unable to search and find supporting files under entities as follows: 1) entities pulled in by the crawler, 2) entities pulled in by DEV on those instances that had this set up previously. (Not sure if the mime-type search in general is not working, or it does not work with entities pulled out by extensions.)

**Comment by fnocetti** [comment]
Hi @hyebin-bina,
After debugging locally I'm not able to reproduce the issue, but I noted from your screenshot that your search is using curly quotes (`attachments.mimetype:“image/png”`) instead of straight quotes (`attachments.mimetype:"image/png"`) which should be used for this type of searches.
Using your exact query locally yields to the same results you are seeing so it's likely that those quotation marks are causing the problem.
Can you kindly give it a try and let me know if that fixes the problem?

**Comment by hyebin-bina** [comment]
Hi @fnocetti ,

We talked about this over MM, but I am also leaving my comment for reporting purposes! 

You were right - when searching mime-type with curly quotes such as: attachments.mimetype:“application/pdf,“ Uwazi then finds entities that include the word "application," while when searching mime-type with straight quotes, it yields the result that we want. And it was tested in Safari and Chrome.

As we discussed, for users whose default quotes are curly ones, it might be helpful to accept both the curly and straight quotes for mime-type searches!

Thank you again, 
Bina
